doc/classes/Particles2D.xml

```diff
@@ -282,43 +276,62 @@
 			<description>
 			</description>
 		</method>
-	</methods>
-	<members>
+	      </methods>
+	      
+	<members>			  
 		<member name="amount" type="int" setter="set_amount" getter="get_amount">
+		  The amount of particles that will be spawned on each emission
 		</member>
 		<member name="draw_order" type="int" setter="set_draw_order" getter="get_draw_order" enum="Particles2D.DrawOrder">
+		  The particle list's drawing order 
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
+		  If true the emitter is currently emitting, if false it is not
 		</member>
 		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
+		  The length of intervals between particle emissions, which can range from 0 to 1
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+		  The particle system's targeted FPS
 		</member>
 		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
+		  If true the fractional delta (time between each frame) will be enabled, if false it will not
```

Comment:
Should be "If [code]true[/code]" per style guide. Remove ", if false it will not"